### ISSUE TITLE
What is the meaning of the 2-th power of the variable 'Win' ? 

### ISSUE THREAD

**Issue by cjfcsjt** [yiqun-wang/MGCN#2]
Hi! Thanks for your great work!
I'm a bit confused with the code in 'MGCN_FAUST.py'
`for k in range(31, -1, -(32//(K-1))):
            Win = torch.mm(torch.mm(torch.mm(V, torch.diag(clk[:, k])), torch.t(V)), torch.diag(torch.squeeze(A))) ** 2
            Win = torch.nn.functional.normalize(Win, p=2, dim=0) ** 2
            list.append(Win)
        Win = torch.stack(list, dim=0)`
It seems that it calculates the wavelet basis, but I do not follow the meaning of the 2-th power of the variable 'Win', Could you explain it please?

**Comment by yiqun-wang** [comment]
Hi, I just want to make the wavelet normalized, and the sum of the normalized wavelet is 1.
